corebehrt/functional/cohort_handling/combined_outcomes.py

```
@@ -0,0 +1,113 @@
+
+def create_empty_results_df():
+    """Create an empty df with columns: PID_COL, TIMESTAMP_COL, ABSPOS_COL"""
+    return pd.DataFrame(columns=[PID_COL, TIMESTAMP_COL, ABSPOS_COL])
+
+
+def check_combination_args(args: dict):
+    if PRIMARY not in args:
+        raise ValueError(f"{PRIMARY} must be defined in the combinations dictionary")
+    if SECONDARY not in args:
+        raise ValueError(f"{SECONDARY} must be defined in the combinations dictionary")
+    timestamp_source = args.get(TIMESTAMP_SOURCE, None)
```

Ruff (0.8.2)

Use `args.get(TIMESTAMP_SOURCE)` instead of `args.get(TIMESTAMP_SOURCE, None)`

Replace `args.get(TIMESTAMP_SOURCE, None)` with `args.get(TIMESTAMP_SOURCE)`

(SIM910)